packages/suite/src/components/suite/labeling/WalletLabeling.tsx

```diff
@@ -13,7 +13,7 @@ export const WalletLabeling = ({ device, shouldUseDeviceLabel }: WalletLabelling
     const { walletLabel } = useSelector(state => selectLabelingDataForWallet(state, device.state));
 
     let label: string | undefined;
-    if (device.metadata.status === 'enabled' && walletLabel) {
+    if (walletLabel) {
```

Comment:
the condition for metadata enabled just become useless here, right? There should be no way of having walletLabel defined if metadata are not enabled. Am I right?

Reply:
right now, I believe so

It could have happend before af3b19b02addef2f2ebb2cd038dd7d4f24c03d36